File: PIPromptClases/C#Clases/Noticia.cs
public class Noticia
    {
        public int NoticiaId { get; set; }
        public string Autor { get; set; }
        public string Descripcion { get; set; }
        public string Imagen { get; set; }
        public string Nombre { get; set; }

        public Noticia() { }

        public Noticia(int noticiaId, string autor, string descripcion, string imagen, string nombre)
        {
            NoticiaId = noticiaId;
            Autor = autor;
            Descripcion = descripcion;
            Imagen = imagen;
            Nombre = nombre;
        }
    }

File: frontend-android/PrototipadoComunidades/proyectobase2425/app/src/main/java/com/pmdm/proyectobase2425/models/Noticia.kt
package com.pmdm.proyectobase2425.models

data class Noticia(
    var noticiaId: Int,
    var autor: String,
    var descripcion: String,
    var imagen: String,
    var nombre: String
) {
    constructor() : this(
        noticiaId = 0,
        autor = "",
        descripcion = "",
        imagen = "",
        nombre = ""
    )
}